src/tiler/server.js

```diff
@@ -55,12 +55,20 @@ var interactivity = {
         tableName = req.params.table;
         stream_order = 0;  // All streams
 
-        if (zoom <= 5) {
-            stream_order = 7;
-        } else if (zoom <= 6) {
-            stream_order = 6;
-        } else if (zoom <= 8) {
-            stream_order = 5;
+        if (tableName === tables.drb_streams_v2) {
+            drb_zoom_levels = {
```

Comment:
I incorrectly assumed 17 was the max zoom; I added a mapping for 18 as well as a check for the zoom level.